vcpkg/users/vcpkg-json.md

```diff
@@ -0,0 +1,395 @@
+---
+title: vcpkg.json Reference
+description: Reference documentation for the vcpkg.json file format.
+ms.date: 11/30/2022
+---
+# vcpkg.json Reference
+
+For an overview of using manifests with vcpkg, see [Manifest mode](manifests.md).
+
+Manifests are strict [JSON](https://www.json.org) documents. They cannot contain C++-style comments (`//`) nor trailing commas. However you can use field names that start with `$` to write your comments in any object that has a well-defined set of keys. These comment fields are not allowed in any objects which permit user-defined keys (such as `"features"`).
+
+The latest JSON Schema is available at [vcpkg.schema.json](https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json). IDEs with JSON Schema support such as Visual Studio and Visual Studio Code can use this file to provide IntelliSense and syntax checking. For most IDEs, you should set `"$schema"` in your `vcpkg.json` to this URL (like the above example).
```

Comment:
Suggestion: Do not use the word "IntelliSense" here since that is only applicable to VS & VS Code. Describe what is meant by IntelliSense here. Syntax checking is one thing, autocompletion and syntax colorization could be additional features.